### ISSUE TITLE
scale in UsdUVTexture not working as I expect

### ISSUE THREAD

**Issue by drwave** [PixarAnimationStudios/USD#988]
### Description of Issue
Per the discussion in

https://graphics.pixar.com/usd/docs/UsdPreviewSurface-Proposal.html 

re: "Roughness vs Glossiness”, I noticed that I wasn’t getting the response I expected.

### Steps to Reproduce

So I wrote a simple example with a scale set to 0:

```python
      def Shader "diffuseTexture"
       {
           uniform token info:id = "UsdUVTexture"
           asset inputs:file = @RedGradientLowerLeftFull.jpg@
           float2 inputs:st.connect = </ShadeTest/boardMat/stReader.outputs:result>
           float4 inputs:scale = (0, 0, 0, 0)
           float3 outputs:rgb
       }
```
In usdview I see the original texture, I expected it to be black. I even switched over to use RenderMan, and I also see the original texture not black, so now I feel like I’m just not understanding how this is supposed to work.
[scaleZero.zip](https://github.com/PixarAnimationStudios/USD/files/3672037/scaleZero.zip)

**Comment by spiffmon** [comment]
Hey @drwave ,
   The problem for RenderMan was what I suspected, *but* we addressed it some time back.  So if you try the dev branch, or wait for the 19.11 release, you should get correct behavior in hdPrman, at least.
   I'll leave this Issue open for the Storm behavior, though.

**Comment by drwave** [comment]
That's great news. I will check when we can get those fixes integrated here for the pram stuff.

**Comment by jtran56** [comment]
Filed as internal issue #USD-5600.

**Comment by sunyab** [comment]
This should be fixed by 86c9446 in the 20.08 release. Closing this out, thanks!
